src/item.cpp

```diff
@@ -3331,6 +3331,11 @@ void item::final_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
             }
         }
     }
+    if( get_option<bool>( "ENABLE_ASCII_ART_ITEM" ) ) {
+        for( const std::string line : type->ascii_picture ) {
```

Comment:
This line makes an unnecessary copy.
```suggestion
        for( const std::string &line : type->ascii_picture ) {
```